### DISCUSSION TITLE
Добавление доменов в список "Россия outside"

### DISCUSSION THREAD

**Discussion by itdoginfo** [itdoginfo/allow-domains#2] [General]
Здесь добавляются домены сервисов, которые блокируют запросы с зарубежных IP-адресов и доступны только с российских подсетей.

Одиночные домены указывайте как есть. Если доменов несколько, то используйте список:
```
domain1.example
domain2.example
```

Для списка доменов одного сервиса используйте форматирование:
```
# domain.example
domain.example
subdomain.domain.example
cdn.domain.example
```

**Comment by samas77** [comment]
#mosenergosbyt.ru
mosenergosbyt.ru
my.mosenergosbyt.ru

**Reply by itdoginfo** [reply]
Done

**Comment by samas77** [comment]
#mos.ru
mos.ru
login.mos.ru
my.mos.ru

**Reply by itdoginfo** [reply]
Он точно недоступен из вне? Я проверял, и сам сайт и ЛК открывались через VPN европейский

**Reply by samas77** [reply]
Через раз, они видимо регулярно гайки закручивают, в частности приложение перестает работать.

**Reply by itdoginfo** [reply]
Понял, добавил. Спасибо

**Comment by samas77** [comment]
#avtodor-tr.ru
avtodor-tr.ru
auth-lkmp.avtodor-tr.ru
lk.avtodor-tr.ru
pays.avtodor-tr.ru

**Reply by itdoginfo** [reply]
Done

**Comment by samas77** [comment]
#ddos-guard.net
AS57724

**Reply by itdoginfo** [reply]
AS в домены не добавить

**Reply by samas77** [reply]
Ну тогда и ладно, это перестраховка .

**Comment by GvozdevLeonid** [comment]
Для кинопоиска нужно добавить только graphql.kinopoisk.ru 
Остальные домены не нужны

**Reply by itdoginfo** [reply]
Хорошо, оставил только его. Но dnsmasq всё-равно будет по wildcard работать

**Reply by GvozdevLeonid** [reply]
Вот новый список
graphql.kinopoisk.ru widevine-proxy.ott.yandex.ru fairplay-proxy.ott.yandex.ru
Первый как и был, второй добавился для веб версии и tv приставок, последний для приложения кинопоиск ( проверено на iphone и ipad)

**Reply by itdoginfo** [reply]
Ок, добавил, но теперь весь yandex.ru будет ходить

**Reply by GvozdevLeonid** [reply]
Понял. Я могу дать точные ip адреса если это поможет. Я использую pbr-iptables и не испытываю проблем с точными доменами

**Reply by GvozdevLeonid** [reply]
graphql.kinopoisk.ru 77.88.21.175
widevine-proxy.ott.yandex.ru 87.250.251.132
fairplay-proxy.ott.yandex.ru 87.250.251.141

**Reply by ProqreS** [reply]
mobileproxy.passport.yandex.net
Вот это еще можно добавить чтобы работали тв каналы. Но не каждый российский vps провайдер проходит идентификацию у Яндекса.

**Comment by Morrowrus** [comment]
есть ли возможность, чтобы telegram ходил к внутренним сервакам, а не зарубежным? очень долго грузяться стикеры/видео с ключенным sing-box, отсюда сделал вывод, что tg идет через прокси.

**Reply by itdoginfo** [reply]
Вы не туда пишете. Здесь чисто про добавление\удаление доменов. Проверяйте маршруты.

**Reply by Morrowrus** [reply]
Почитал [статью ](https://itdog.info/tochechnaya-marshrutizaciya-po-domenam-na-routere-s-openwrt/) - все получилось. спасибо!)

**Comment by Kazanovasgh** [comment]
yandex.net - связано с яндекс музыкой (именно *.net)

**Reply by itdoginfo** [reply]
Done

**Comment by klim123123** [comment]
leroymerlin.ru
lemanapro.ru

**Reply by itdoginfo** [reply]
Done

**Comment by klim123123** [comment]
pesc.ru

не постоянно блокируется, а рандомно. видимо при каких то атаках.

**Reply by itdoginfo** [reply]
Done

**Comment by Nocturnal-ru** [comment]
Не претендую на истину, но если правильно понимаю, в списках поддерживается wildcard, тогда зачем добавлять кучу доменов третьего уровня, если добавлен уже домен второго уровня, речь в первую очередь о
`mos.ru`
`ozon.ru`
`rzd.ru`

**Reply by itdoginfo** [reply]
Это сделано было сделано для будущего. Для списка без wildcard, пока таких нет

**Comment by muradurm** [comment]
rosreestr.gov.ru
fssp.gov.ru
xn--90aijkdmaud0d.xn--p1ai

**Reply by itdoginfo** [reply]
Done

**Comment by generalmarriott** [comment]
#mosreg.ru
mosreg.ru
pos.gosuslugi.ru
dobrodel.mosreg.ru

#genotek.ru
    lk.genotek.ru
    lk2-back.genotek.ru
    bitrix.genotek.ru
    back.genotek.ru

**Reply by itdoginfo** [reply]
Добавил только домены второго уровня. Скажите если нужен только третий

**Reply by generalmarriott** [reply]
Спасибо, но что-то оно не добавилось...  вернее только мосрег вижу, а генотека нет. 
https://raw.githubusercontent.com/itdoginfo/allow-domains/main/Russia/outside-dnsmasq-nfset.lst

**Reply by generalmarriott** [reply]
в общем немного доработал под себя и сделал себе свой собственный список с блэкджеком и моментальным добавлением. 
Не хочу показаться неблагодарным. Работа очень огромная, спасибо большое! Просто для меня важно быстренько добавить свои домены, а то потом забуду. то мне надо было что-то сделать  :)

**Comment by pzvezdin** [comment]
lkfl2.nalog.ru
client.reso.ru

**Reply by itdoginfo** [reply]
Идентично

**Comment by muradurm** [comment]
gu-st.ru

Без этого домена перестали работать госуслуги.

**Reply by itdoginfo** [reply]
Done

**Comment by gosro** [comment]
kali.org

**Reply by itdoginfo** [reply]
Это же не российский сайт. Он где-то недоступен?

**Comment by kyzima-spb** [comment]
gorzdrav.spb.ru

**Reply by itdoginfo** [reply]
Done

**Comment by ArtemDNK** [comment]
emex.ru

**Reply by itdoginfo** [reply]
Done

**Comment by Randall44** [comment]
ozon.ru при подключении списка Russia outside не открывается из РБ, в частности приложение. Если список не применять, то все работает нормально.

**Reply by Graandfather228** [reply]
Нужно добавить api.ozon.ru должно сработать, попробуйте (это домен приложения)

**Comment by Ross-dot11** [comment]
Добрый день, а можно добавить вот это 

consultant.ru и cloud.consultant.ru

**Reply by itdoginfo** [reply]
done

**Comment by spgsroot** [comment]
sberbank.ru
sber.ru
sbrf.ru
sberbank.com
sberpay.ru
vtb.ru
alfabank.ru
alfabank.st
tinkoff.ru
tbank.ru
t-bank.ru
gazprombank.ru
gpb.ru
psbank.ru
psb.ru
rshb.ru
sovcombank.ru
raiffeisen.ru
gosuslugi.ru
nalog.ru
nalog.gov.ru
cbr.ru
mironline.ru
nspk.ru
wildberries.ru
wb.ru
wbbasket.ru
ozon.ru
ozonusercontent.com
ozonbank.ru
avito.ru
avito.st
yandex.ru
yandex.net
yandex.com
ya.ru
yastatic.net
kinopoisk.ru
vk.com
vk.ru
userapi.com
vk-cdn.net
vkuser.net
mail.ru
imgsmail.ru
mycdn.me
ok.ru
max.ru
oneme.ru
2gis.ru
2gis.ua
2gis.kz
2gis.by
2gis.com
rutube.ru

**Reply by itdoginfo** [reply]
Часть есть. Сверьтесь со списком.

**Comment by sergeyhatunzev** [comment]
https://5ka.ru/ выдает  Проблемы со связью.
Проверьте настройки интернета и VPN,  механизм проверки скрыт в специальном запросе на домен https://x5m.x5.ru/common/api/gatekeeper/public/v1/evaluate c девайс id если просто открыть в браузере дает 200
создал пост  вместо написания сюда по ошибке удалить не смог

**Comment by sergeyhatunzev** [comment]
https://magnit.ru/

**Reply by itdoginfo** [reply]
done

**Comment by Anderhar** [comment]
lamoda.ru

**Comment by lexfrei** [comment]
Сверился: из тех 55 доменов от 11 мая не заехал ни один. Четыре совпадения (gosuslugi.ru, nalog.ru, ozon.ru, yandex.net) лежали в списке ещё до того комментария, а после него добавились только consultant.ru и magnit.ru, и то из других заявок. Остальные висят в очереди или отклонены? Если простыня в 55 строк неудобная, могу раскидать по одному сервису.

От себя ещё пара:

```
# avito.ru
avito.ru
avito.st

# vseinstrumenti.ru
vseinstrumenti.ru
```

Оба блокируют молча — ни 403, ни заглушки, страница просто висит и не догружается. Зацепиться автоматике не за что, поэтому сами они и не всплыли.
